docs/source/building_an_experiment.rst

```diff
@@ -4,14 +4,227 @@ Building an Experiment
+    needed to write the beam_order list. This is like a mapping of beam number (list
+    index) to beam direction off boresight.
+
+beam_order
+    beam numbers written in order of preference, one element in this list corresponds to
+    one integration period. Can have lists within the list, resulting in multiple beams
+    running simultaneously in the averaging period, so imaging. A beam number of 0 in
+    this list gives us the direction of the 0th element in the beam_angle list. It is
+    up to the writer to ensure their beam pattern makes sense. Typically beam_order is
+    just in order (scanning W to E or E to W, ie. [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10,
+    11, 12, 13, 14, 15]. You can list numbers multiple times in the beam_order list,
+    for example [0, 1, 1, 2, 1] or use multiple beam numbers in a single
+    integration time (example [[0, 1], [3, 4]], which would trigger an imaging
+    integration. When we do imaging we will still have to quantize the directions we
+    are looking in to certain beam directions.
+
+scanbound
+    A list of seconds past the minute for integration times in a scan to align to.
+
+clrfrqrange
+    range for clear frequency search, should be a list of length = 2, [min_freq, max_freq]
+    in kHz.
+
+txfreq
+    transmit frequency, in kHz. Note if you specify clrfrqrange it won't be used.
+
+rxfreq
+    receive frequency, in kHz. Note if you specify clrfrqrange or txfreq it won't be used. Only
+    necessary to specify if you want a receive-only slice.
+
+wavetype
+    string for wavetype. The default is SINE. Any other wavetypes not currently supported but
+    possible to add in at later date.
+
+iwavetable
+    a list of numeric values to sample from. The default is None. Not currently supported
+    but could be set up (with caution) for non-SINE.
+
+qwavetable
+    a list of numeric values to sample from. The default is None. Not currently supported
+    but could be set up (with caution) for non-SINE.
+
+seqoffset
+    offset in us that this slice's sequence will begin at, after the start of the sequence.
+    This is intended for PULSE interfacing, when you want multiple slice's pulses in one sequence
+    you can offset one slice's sequence from the other by a certain time value so as to not run both
+    frequencies in the same pulse, etc.
+
+comment
+    a comment string that will be placed in the borealis files describing the slice.
+
+acf
+    flag for rawacf and generation. The default is False.
+
+xcf
+    flag for cross-correlation data. The default is True if acf is True, otherwise False.
+
+acfint
+    flag for interferometer autocorrelation data. The default is True if acf is True, otherwise
+    False.
+
+range_sep
+    a calculated value from pulse_len. If already set, it will be overwritten to be the correct
+    value determined by the pulse_len. Used for acfs. This is the range gate separation,
+    in azimuthal direction, in km.
```

Comment:
Wouldn't this be the range gate separation in the radial direction, not the azimuthal direction? Range gate separation in the azimuthal direction is set by the beam_angle list